lib/team.rb

```diff
@@ -20,24 +14,14 @@ def self.find_team_info(id)
     team_info
   end
 
-  def self.from_csv(file_path)
-    csv = CSV.read("#{file_path}", headers: true, header_converters: :symbol)
-    @@all = csv.map do |row|
-      Team.new(row)
-    end
-  end
-
-
-
   def self.count_of_teams
     all.length
   end
 
-  attr_reader :game_teams, :team_id, :franchise_id, :team_name, :abbreviation, :stadium, :link, :team_name
+  attr_reader :team_id, :franchise_id, :team_name, :abbreviation, :stadium, :link, :team_name
 
 
   def initialize(team_info)
-    @game_teams = game_teams
```

Comment:
it appeared that this wasn't being utilized at all. Deleted and caused no errors or concern.

Reply:
that was probably from when my methods were in team class! Absolutely not needed haha